usr/local/www/system.php

```diff
@@ -511,6 +512,32 @@ function is_timezone($elt) {
 					</strong>
 				</td>
 			</tr>
+			<tr>
+				<td width="22%" valign="top" class="vncell">&nbsp;</td>
+				<td width="78%" class="vtable">
+					<select name="theme_dashboard">
+						<?php
+							$files = return_dir_as_array("/usr/local/www/themes/");
+							foreach($files as $f):
+								if ((substr($f, 0, 1) == "_") && !isset($config['system']['developer']))
+									continue;
+								if ($f == "CVS")
+									continue;
+								$curtheme = "pfsense";
```

Comment:
When I tried this, I immediately got "pfsense" theme on the dashboard, when my overall theme was set to pfsense_ng. I think that was even before selecting a them here and saving.
Perhaps $curtheme here needs to default to $config['theme']
 And the code up in index.php should only happen if isset($config['theme_dashboard']) - at the moment that code executes when 'theme_dashboard' has not been set, which I think is why on first use $config['theme'] ends up null, and then I guess something later uses 'pfsense' as the theme when the theme is blank.